articleEditor/editor.js

```diff
@@ -166,48 +178,39 @@ function addArticleToStorage(article) {
 }
 
 function ShowArticles(arrayArticle) {
-
     document.querySelector(`.cardsContainer`).innerHTML = "";
 
     for (let i = 0; i < arrayArticle.length; i++) {
         createCard(arrayArticle[i]);
-        document.querySelectorAll(`.card-text`)[i].setAttribute(`data-article-number`, i);
+        document.querySelector(`.cardsContainer`).querySelectorAll(`.card-text`)[i]
```

Comment:
Totally bad idea. You call double selector each time in array, instead to use local variable e.q.:
`let cardTexts = document.querySelector(`.cardsContainer .card-text`);`